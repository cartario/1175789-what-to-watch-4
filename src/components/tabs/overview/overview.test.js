import React from "react";
import renderer from "react-test-renderer";
import Overview from "./overview.jsx";

const currentMovie = {
  rating: 1,
  scoresCount: 1,
  description: ``,
  director: ``,
  starring: [],
};

describe(`Overview-test`, () => {
  it(`should render Overview`, () => {
    const tree = renderer
      .create(
          <Overview
            currentMovie = {currentMovie}
          />
      )
      .toJSON();

    expect(tree).toMatchSnapshot();
  });
});
