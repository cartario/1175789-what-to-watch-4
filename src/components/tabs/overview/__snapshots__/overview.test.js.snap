// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Overview-test should render Overview 1`] = `
Array [
  <div
    className="movie-rating"
  >
    <div
      className="movie-rating__score"
    >
      1
    </div>
    <p
      className="movie-rating__meta"
    >
      <span
        className="movie-rating__level"
      >
        Bad
      </span>
      <span
        className="movie-rating__count"
      >
        1
         ratings
      </span>
    </p>
  </div>,
  <div
    className="movie-card__text"
  >
    <p>
      
    </p>
    <p
      className="movie-card__director"
    >
      <strong>
        Director: 
        
      </strong>
    </p>
    <p
      className="movie-card__starring"
    >
      <strong>
        Starring: 
      </strong>
    </p>
  </div>,
]
`;
